app/src/main/java/com/example/areumdap/UI/Chat/ChatViewModel.kt

```
@@ -0,0 +1,113 @@
+                val reply = repo.ask(trimmed)
+
+                _messages.value = _messages.value
+                    .filterNot { it.id == typingId } +
+                        ChatMessage(
+                            id = "ai_${System.currentTimeMillis()}",
+                            sender = Sender.AI,
+                            text = reply,
+                            time = System.currentTimeMillis(),
+                            status = Status.SENT
+                        )
+            } catch (e: Exception){
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)